docs/technical_reference/docfx_exe_cli_reference.md

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
Consider:
- `-` over `_` as the URL concat char
- docfx.exe -> docfx since we now support linux and mac and there is no `docfx.exe` there
- move to `reference` folder to match TOC name

*docs/technical_reference/docfx_exe_cli_reference.md* -> *docs/reference/docfx-cli-reference.md*

Reply:
- Replaced '_' with '-' in Markdown file names
- Removed '.exe' throughout in both documentation and .md filenames
- Corrected 'technical_reference' folder name to 'reference'